README.md

```diff
@@ -112,7 +112,7 @@ Join the [Official Homebridge Discord](https://discord.gg/Cmq8a44) community and
 
 ## License
 
-Copyright (C) 2017-2022 oznu
+Copyright (C) 2017-2023 homebridge
```

Comment:
Add a new line above while retaining the existing ones 😀

Reply:
Like this ?

```
Copyright (C) 2023 homebridge
Copyright (C) 2017-2022 oznu
```

Reply:
LGTM